tsconfig.json

```diff
@@ -1,19 +1,17 @@
 {
-  "ts-node": {
-    "esm": false // Should be changed to true once stable
-  },
   "include": ["src/**/*"],
   "compilerOptions": {
     "baseUrl": "./src",
     "outDir": "dist",
-    "target": "es2020",
-    "lib": ["es2020"],
-    "module": "CommonJS", // Should be changed to ES2020 once stable
-    "moduleResolution": "Node",
+    "module": "node16",
+    "moduleResolution": "node16",
```

Comment:
:thinking: Any reason not to set this to `NodeNext`?

Reply:
because CI is failing:
```
 FAIL  test/v0.test.ts
  ● Test suite failed to run

    SyntaxError: The requested module '#crypto' does not provide an export named 'subtle'

      at async Runtime.linkAndEvaluateModule (node_modules/jest-runtime/build/index.js:708:5)
```

so I assume it's from `@octokit/auth-app`
They are saying at the docs to use node16 but it looks like that's not the issue

Reply:
We are using a very old CI image, could you try updating it?
https://github.com/digitalfabrik/app-toolbelt/blob/bf54649ba4bf06a970f4a9dff95b8c706d67c86e/.circleci/config.yml#L6
For example with
https://github.com/digitalfabrik/integreat-app/blob/ade679252b4c9591406f701c4e478204f5e5284a/.circleci/src/jobs/check.yml#L2C1-L3C29